<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>342</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>10</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="libres.qrc">
    <normaloff>:/lib/fdogicon.png</normaloff>:/lib/fdogicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:Microsoft YaHei;}

#Login{
background-color: rgb(84, 62, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>431</width>
       <height>304</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>425</width>
       <height>330</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#widet{
border-radius: 10px;
border-style:solid;
background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <widget class="QLabel" name="mian_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>431</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
border-top-left-radius:10px;
border-top-right-radius:10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>129</y>
        <width>431</width>
        <height>200</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>431</width>
        <height>200</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="register_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>299</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#register_btn{
color: rgb(149, 149, 149);
}
#register_btn:hover{
color: rgb(120, 120, 120);
}</string>
      </property>
      <property name="text">
       <string>注册账号</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>227</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="minimumSize">
          <size>
           <width>78</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#checkBox{
color: rgb(149, 149, 149);
}
#checkBox:hover{
color: rgb(120, 120, 120);
}</string>
         </property>
         <property name="text">
          <string>自动登录</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="minimumSize">
          <size>
           <width>78</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#checkBox_2{
color: rgb(149, 149, 149);
}
#checkBox_2:hover{
color: rgb(120, 120, 120);
}</string>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>57</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_2{
color: rgb(149, 149, 149);
}
#pushButton_2:hover{
color: rgb(120, 120, 120);
}</string>
         </property>
         <property name="text">
          <string>找回密码</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox 
{ 
font-size:15px;
}
QComboBox QAbstractItemView::item
{
min-height: 50px; 
min-width: 60px; 

}
::drop-down{
	border-style:solid;
	border-width:1px;
	width:20px ;
	background-color: rgb(255, 255, 255);
	border-bottom-color: rgb(129, 129, 129);
}
::down-arrow{
border-image: url(:/lib/jietou1.png);
}</string>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
      <property name="insertPolicy">
       <enum>QComboBox::NoInsert</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>134</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width:2px;
border-style:solid;
border-radius: 8px;
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="libres.qrc">
        <normaloff>:/lib/zaixian.png</normaloff>:/lib/zaixian.png</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>79</y>
        <width>72</width>
        <height>72</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-style:solid;
border-radius: 36px;
background-color: rgba(255, 255, 255,200);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>204</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>12</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:solid;
border-width:1px;
border-top-color: rgba(255, 255, 255, 0);
border-right-color: rgba(255, 255, 255, 0);
border-left-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(129, 129, 129);


</string>
      </property>
      <property name="placeholderText">
       <string>Fdog号码/邮箱/手机号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>200</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>14</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:solid;
border-width:1px;
border-top-color: rgba(255, 255, 255, 0);
border-right-color: rgba(255, 255, 255, 0);
border-left-color: rgba(255, 255, 255, 0);
border-bottom-color: rgb(129, 129, 129);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>277</y>
        <width>221</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{
background-color: rgb(99, 188, 255);
color: rgb(255, 255, 255);
border-radius: 5px;
}
#pushButton:hover{
background-color: rgb(99, 170, 255);
color: rgb(255, 255, 255);
border-radius: 5px;

}
#pushButton:pressed{
background-color: rgb(99, 188, 255);
color: rgb(255, 255, 255);
border-radius: 5px;
padding-left:2px;
padding-top:2px;
}
</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>0</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolButton_4">
         <property name="minimumSize">
          <size>
           <width>83</width>
           <height>39</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>83</width>
           <height>39</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:solid;
background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="libres.qrc">
           <normaloff>:/lib/mainicon.png</normaloff>:/lib/mainicon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:solid;
background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="libres.qrc">
           <normaloff>:/lib/zuixiaohua.png</normaloff>:/lib/zuixiaohua.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-style:solid;
background-color: rgba(255, 255, 255, 0);</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="libres.qrc">
           <normaloff>:/lib/guanbi.png</normaloff>:/lib/guanbi.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>82</y>
        <width>66</width>
        <height>66</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/lib/icon2505897252.png);
border-width:0px;
border-style:solid;
border-radius:32px;
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>290</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:solid;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="libres.qrc">
        <normaloff>:/lib/erweima.png</normaloff>:/lib/erweima.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <zorder>mian_label</zorder>
     <zorder>label_2</zorder>
     <zorder>register_btn</zorder>
     <zorder>horizontalLayoutWidget_2</zorder>
     <zorder>comboBox</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>lineEdit</zorder>
     <zorder>pushButton</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>toolButton_3</zorder>
     <zorder>toolButton_5</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>register_btn</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>toolButton_3</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="libres.qrc"/>
 </resources>
 <connections/>
</ui>
